### DISCUSSION TITLE
Hostname at WATCHTOWER_NOTIFICATION_TEMPLATE?

### DISCUSSION THREAD

**Discussion by jorburmar** [containrrr/watchtower#800] [Q&A] [answered]
is there any way to put the hostname in the template?
i try {{.hostname}} and {{os.hostname}} with no results.

**Comment by piksel** [comment, marked as answer]
No, only the `logrus.Entry` fields are available in the template right now. But yeah, there are a couple of fields like that, which could be really useful to add!

**Reply by jorburmar** [reply]
thanks for the reply ;)
